### ISSUE TITLE
Lua error on discovering Bontom's location (in BfW 1.16)

### ISSUE THREAD

**Issue by cooljeanius** [nemaara/A_New_Order#39]
I dunno what to do about this... 
<img width="575" alt="Screen Shot 2021-12-13 at 23 11 41" src="https://user-images.githubusercontent.com/1388660/145931872-5b57609a-ba6a-48a2-94e2-d1b932f6c912.png">

(This is S13, "Scouting", btw... possibly related to issue #24?)

**Comment by cooljeanius** [comment]
ok so cooljeanius/A_New_Order@74dd61c fixes it on my end; I'm going to leave this open until it's merged here, though

**Comment by cooljeanius** [comment]
> ok so [cooljeanius/A_New_Order@74dd61c](https://github.com/cooljeanius/A_New_Order/commit/74dd61c) fixes it on my end; I'm going to leave this open until it's merged here, though
> 
> (I'll be doing a separate PR that includes it after #36 is merged)

the "separate PR that includes it" is PR #43 now
